tests/external_botocore/test_bedrock_chat_completion.py

```diff
@@ -209,22 +218,39 @@ def test_bedrock_chat_completion_disabled_settings(set_trace_info, exercise_mode
 )
 @background_task()
 def test_bedrock_chat_completion_error_invalid_model(bedrock_server, set_trace_info):
-    set_trace_info()
-    add_custom_attribute("conversation_id", "my-awesome-id")
-    with pytest.raises(_client_error):
-        bedrock_server.invoke_model(
-            body=b"{}",
-            modelId="does-not-exist",
-            accept="application/json",
-            contentType="application/json",
-        )
+    @validate_custom_events(chat_completion_invalid_model_error_events)
+    @validate_error_trace_attributes(
+        "botocore.errorfactory:ValidationException",
+        exact_attrs={
+            "agent": {},
+            "intrinsic": {},
+            "user": {
```

Comment:
We should be checking for completion_id here too.

Reply:
Totally agree with you. I had added that in previously but given the `completion_id` is UUID and will differ on each test run, I set this to `None` but this validator does accept that behavior. It just causes the whole test to fail expecting the ID to be the specific UUID for that test run.